tools/devpack.ps1

```diff
@@ -1,10 +1,38 @@
-﻿param(
+param(
     [Parameter(Mandatory=$false)]
     [Switch]
     $E2E
 )
 
 
+function Get-LatestPackage-Version([string] $packageName, [string] $packagePath)
+{
+    $packageVersion = $null
+
+    Get-ChildItem "$packagePath" -Name |
+    Foreach-Object {
+        if ($_ -Match "$packageName")
+        {
+            # Converts the nupkg file name to a version
+            $currentVersion = $_ -replace "$packageName.", ""
+            $currentVersion = $currentVersion -replace ".nupkg", ""
+
+            # Gets the highest version
+            if ($currentVersion -gt $packageVersion)
```

Comment:
Is this doing string comparison? There's some weirdness with that where something like `1.0` will be sorted as higher than `10.0`. I believe this package gives you a way to sort with nuget's semantic versioning: https://www.nuget.org/packages/NuGet.Versioning.

It's probably not a big deal, but when it sorts incorrectly locally it'll be really confusing.